packages/flutter/lib/src/widgets/table.dart

```diff
@@ -145,19 +145,44 @@ class Table extends RenderObjectWidget {
        }()),
        assert(() {
          if (children.isNotEmpty) {
-           final int cellCount = children.first.children.length;
```

Comment:
Since TableCell.non is used in place of hidden cells, shouldn't this assertion logic still work? A TableCell for every cell?